extensions/skyportal/skyportal/handlers/api/alert.py

```diff
@@ -1063,3 +1063,120 @@ async def get(self, objectId: str = None):
         except Exception:
             _err = traceback.format_exc()
             return self.error(f"failure: {_err}")
+
+
+class ZTFAlertsByCoordsHandler(ZTFAlertHandler):
+    @auth_or_token
+    def get(self):
+        """
+        ---
+        single:
+          description: Query Kowalski for alerts near specified coords
+          parameters:
+            - in: query
+              name: ra
+              required: true
+              schema:
+                type: float
+              description: RA in degrees
+            - in: query
+              name: dec
+              required: true
+              schema:
+                type: float
+              description: Dec. in degrees
+            - in: query
+              name: radius
+              required: true
+              schema:
+                type: float
+              description: Max distance from specified (RA, Dec) in degrees
+          responses:
+            200:
+              description: retrieved alert(s)
+              content:
+                application/json:
+                  schema:
+                    allOf:
+                      - $ref: '#/components/schemas/Success'
+                      - type: object
+                        properties:
+                          data:
+                            type: array
+                            items:
+                              type: object
+            400:
+              content:
+                application/json:
+                  schema: Error
+        """
+        ra = self.get_query_argument("ra", None)
+        dec = self.get_query_argument("dec", None)
+        radius = self.get_query_argument("radius", None)
+        if ra is None:
+            return self.error("Missing required parameter: ra")
+        if dec is None:
+            return self.error("Missing required parameter: dec")
+        if radius is None:
+            return self.error("Missing required parameter: radius")
+        try:
+            ra = float(ra)
+            dec = float(dec)
+            radius = float(radius)
```

Comment:
Let's put in a check that the radius is not more than, say 1 degree - noob protection.

Reply:
Should we still ensure radius is no more than one degree if we're limiting the number of results in the query?